client.go

```diff
@@ -30,9 +30,9 @@ type RunTokenResponse struct {
 }
 
 type FunctionRunResponse struct {
-	AppUUID string `json:"appUuid"`
-	RunID   string `json:"runId"`
-	Result  []byte `json:"result"`
+	AppUUID string                 `json:"appUuid"`
+	RunID   string                 `json:"runId"`
+	Result  map[string]interface{} `json:"result"`
```

Comment:
```suggestion
	Result  map[string]any `json:"result"`
```
Since go 1.18 you can replace any instance of `interface{}` with the `any` alias

Reply:
Nice, thanks much cleaner!